MyDataHelpsKit.xcodeproj/project.pbxproj

```diff
@@ -349,7 +358,7 @@
 			);
 			outputPaths = (
 			);
-			runOnlyForDeploymentPostprocessing = 0;
+			runOnlyForDeploymentPostprocessing = 1;
```

Comment:
Finally learned how to fix the "Automatic preview updating paused" problem with SwiftUI previews: [custom build scripts must have "For install builds only" checked](https://stackoverflow.com/a/73518557/795339). This project has such a custom build script, for some auto-generated code in SDKVersion.swift and the Podspec file.

Reply:
Ah, good to know!